repositories.bzl

```diff
@@ -63,6 +64,8 @@ airy_jvm_deps = [
     "org.springframework:spring-websocket:5.3.6",
     "org.springframework.security:spring-security-core:5.4.6",
     "org.rocksdb:rocksdbjni:5.18.3",
+    "com.segment.analytics.java:analytics-core:3.1.3",
```

Comment:
@chrismatix This fix the issue with the `tools/tracking` library. I don't know how it was working before any ideas?

Reply:
@juan-sebastian What exactly wasn't working? 🤔

Reply:
After doing
```sh
$ bazel run @unpinned_maven//:pin
```
and updating the maven dependence the `//lib/java/tracking` build was not working anymore.